core/src/main/java/apoc/export/util/BatchTransaction.java

```diff
@@ -53,10 +53,13 @@ private Transaction beginTx() {
         return gdb.beginTx();
     }
 
+    public void lastCommit() {
+        tx.commit();
+    }
+    
     @Override
     public void close() {
         if (tx!=null) {
-            tx.commit();
```

Comment:
no rimettilo stai cambiando completamente il comportamento delle API che potrebbero essere usate in un altro punto

Reply:
Ok cambiato, ora ho rimosso il try-resources dai `BatchTransaction` perché quando va in errore non si dovrebbe chiamare `tx.commit()`